go.mod

```diff
@@ -422,3 +422,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.14.2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/rancher/shepherd => /Users/ugur/dev/src/github.com/caliskanugur/shepherd
```

Comment:
Should this be in here?

Reply:
That's a great question. Yes, it shouldn't be there I'll change it shortly while doing sanity checks, once Shepherd is merged, I'll update it again.